src/scripts/main.js

```diff
@@ -354,7 +354,47 @@ const people = [
   },
 ];
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const table = document.querySelector('tbody');
 
-// write your code here
+people.forEach((person) => {
+  const row = document.createElement('tr');
+
+  for (const key in person) {
+    const cell = document.createElement('td');
+
+    switch (key) {
+      case 'name':
+        cell.textContent = person.name;
+        row.appendChild(cell);
```

Comment:
you can write this line only one time after all cases

Reply:
if i write 'row.appendChild(cell);' after switch-case i have thre extras empty cells